<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetTest</class>
 <widget class="QWidget" name="widgetTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget1" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,20,1,1,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnPreX">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="png.qrc">
              <normaloff>:/png/png toolbar/Kombine_toolbar_007.png</normaloff>:/png/png toolbar/Kombine_toolbar_007.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutX">
          <property name="spacing">
           <number>2</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnNextX">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="png.qrc">
              <normaloff>:/png/png toolbar/Kombine_toolbar_025.png</normaloff>:/png/png toolbar/Kombine_toolbar_025.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="labelXT">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>T(us):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelXA">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>A(mV):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelXValueIndex">
            <property name="text">
             <string>信号读时:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,20,1,1,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnPreXx">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="png.qrc">
              <normaloff>:/png/png toolbar/Kombine_toolbar_007.png</normaloff>:/png/png toolbar/Kombine_toolbar_007.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutXx">
          <property name="spacing">
           <number>2</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>X'</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnNextXx">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="png.qrc">
              <normaloff>:/png/png toolbar/Kombine_toolbar_025.png</normaloff>:/png/png toolbar/Kombine_toolbar_025.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="labelXxT">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>T(us):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelXxA">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>A(mV):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelXxValueIndex">
            <property name="text">
             <string>信号读时:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,20,1,1,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnPreY">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="png.qrc">
              <normaloff>:/png/png toolbar/Kombine_toolbar_007.png</normaloff>:/png/png toolbar/Kombine_toolbar_007.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutY">
          <property name="spacing">
           <number>2</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnNextY">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="png.qrc">
              <normaloff>:/png/png toolbar/Kombine_toolbar_025.png</normaloff>:/png/png toolbar/Kombine_toolbar_025.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="labelYT">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>T(us):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelYA">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>A(mV):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelYValueIndex">
            <property name="text">
             <string>信号读时:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,20,1,1,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnPreYy">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="png.qrc">
              <normaloff>:/png/png toolbar/Kombine_toolbar_007.png</normaloff>:/png/png toolbar/Kombine_toolbar_007.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="layoutYy">
          <property name="spacing">
           <number>2</number>
          </property>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Y'</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnNextYy">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="png.qrc">
              <normaloff>:/png/png toolbar/Kombine_toolbar_025.png</normaloff>:/png/png toolbar/Kombine_toolbar_025.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>2</number>
          </property>
          <item>
           <widget class="QLabel" name="labelYyT">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>T(us):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelYyA">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>A(mV):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelYyValueIndex">
            <property name="text">
             <string>信号读时:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="png.qrc"/>
 </resources>
 <connections/>
</ui>
